rules/retrieve.smk

```diff
@@ -56,6 +56,24 @@ if config["enable"]["retrieve"] and config["enable"].get("retrieve_databundle",
         script:
             "../scripts/retrieve_eurostat_data.py"
 
+    rule retrieve_jrc_idees:
+        output:
+            directory("data/bundle/jrc-idees-2021"),
+        log:
+            "logs/retrieve_jrc_idees.log",
+        retries: 2
+        script:
+            "../scripts/retrieve_jrc_idees.py"
+
+    rule retrieve_ammonia_demand:
+        output:
+            "data/bundle/myb1-2022-nitro-ert.xlsx",
+        log:
+            "logs/retrieve_ammonia_demand.log",
+        retries: 2
+        script:
+            "../scripts/retrieve_ammonia_demand.py"
```

Comment:
I think this can be replaced by

```py
rule build_ammonia_production:
    input:
        usgs=storage("https://d9-wret.s3.us-west-2.amazonaws.com/assets/palladium/production/s3fs-public/media/files/myb1-2022-nitro-ert.xlsx"),
```

```suggestion
```

Can also remove entry here:

https://github.com/PyPSA/pypsa-eur/blob/7a16dd2aa4b215037899514c6147711eae07686f/rules/retrieve.smk#L26

(another one down in the data bundle :partying_face:)